tests/acceptance/features/bootstrap/SearchContext.php

```diff
@@ -52,13 +52,13 @@ class SearchContext implements Context {
 	 * @return void
 	 */
 	public function userSearchesUsingWebDavAPI(
-		string $user,
-		string $pattern,
-		?string	$limit = null,
-		?string $scope = null,
-		?string $spaceName = null,
+		string    $user,
+		string    $pattern,
+		?string   $limit = null,
+		?string   $scope = null,
+		?string   $spaceName = null,
```

Comment:
not single space?
```suggestion
		string  $user,
		string  $pattern,
		?string $limit = null,
		?string $scope = null,
		?string $spaceName = null,
```

Reply:
I think the linter was trying to match with`TableNode $properties = null`

Reply:
ahh, didn't notice that one :+1: